tests/Platforms/AbstractPostgreSQLPlatformTestCase.php

```diff
@@ -19,9 +21,12 @@
 
 abstract class AbstractPostgreSQLPlatformTestCase extends AbstractPlatformTestCase
 {
-    /** @var PostgreSQL94Platform */
+    /** @var PostgreSQL94Platform|PostgreSQL100Platform */
```

Comment:
Why is this needed? `PostgreSQL100Platform` is a subtype of `PostgreSQL94Platform`.

Reply:
Not needed, I just didn't notice that was the case.